src/Tokenizer/Token.php

```diff
@@ -61,9 +61,14 @@ public function __construct($token)
             $this->isArray = true;
             $this->id = $token[0];
             $this->content = $token[1];
-        } else {
+        } elseif (is_string($token)) {
             $this->isArray = false;
             $this->content = $token;
+        } else {
+            throw new \InvalidArgumentException(sprintf(
```

Comment:
that's what happen when you extract not-related, side-off changes from another, big PR...